wmn-data.json

```diff
@@ -998,6 +998,18 @@
         "m_code" : 404,
         "known" : ["dictvm", "sml"],
         "cat" : "social",
+        "valid" : true
+       },        
+       {
+        "name" : "chaturbate",
+        "uri_check" : "https://chaturbate.com/{account}/",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "'s Bio and Free Webcam",
+        "m_string" : "It's probably just a broken link",
+        "m_code" : 302,

```

Comment:
Can you check the `m_code` as I'm only seeing 404 and not 302?

Reply:
@WebBreacher First I got a `404`, tested that with the python script and there was false positives every try, then I put the link through this tool: https://www.webfx.com/tools/http-status-tool 
this tool returned a 302, I then changed the m_code to 302, this gave a good detection via the python script.
But no worries, I can change it to 404 if it's good your end.

Reply:
@WebBreacher  `404 Change`:  Done ✅